docassemble/AssemblyLine/data/questions/al_saved_sessions.yml

```
@@ -4,6 +4,34 @@ code: |
   # Create a placeholder value to avoid playground errors
   al_sessions_snapshot_results = DAEmpty() 
 ---
+initial: True
+code: |
+  ########### Save title and progress to session list ##################
+  if get_config("assembly line",{}).get("update answer set metadata"):
+    # To limit impact on DB, save progress every 3 steps or when progress is 100 +- 1
+    if get_progress() > 99 or int(all_variables(include_internal=True).get('_internal',{}).get('steps',1)) % 3 == 1:
```

```suggestion
    if get_progress() > 99 or int(_internal.get('steps',1)) % 3 == 1:
```
Should be the same thing, it returns the same info in some demo interviews.